plugins/woocommerce/src/Admin/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php

```diff
@@ -0,0 +1,773 @@
+	}
+
+	/**
+	 * Get the template description.
+	 */
+	public function get_description(): string {
+		return __( 'Template for the simple product form', 'woocommerce' );
+	}
+
+	/**
+	 * Get a group block by ID.
+	 *
+	 * @param string $group_id The group block ID.
+	 */
+	public function get_group_by_id( string $group_id ): ?GroupInterface {
+		return $this->get_block( $group_id );
+	}
+
+	/**
+	 * Get a section block by ID.
+	 *
+	 * @param string $section_id The section block ID.
+	 */
+	public function get_section_by_id( string $section_id ): ?SectionInterface {
+		return $this->get_block( $section_id );
+	}
+
+	/**
+	 * Get a block by ID.
+	 *
+	 * @param string $block_id The block block ID.
+	 */
+	public function get_block_by_id( string $block_id ): ?BlockInterface {
+		return $this->get_block( $block_id );
+	}
+
+	/**
+	 * Add a custom block type to this template.
+	 *
+	 * @param array $block_config The block data.
+	 */
+	public function add_group( array $block_config ): GroupInterface {
+		$block = new Group( $block_config, $this->get_root_template(), $this );
+		return $this->add_inner_block( $block );
+	}
+
+	/**
+	 * Adds the general group blocks to the template.
+	 */
+	private function add_general_group_blocks() {
+		$general_group = $this->add_group(
+			[
+				'id'         => 'general',
+				'order'      => 10,
+				'attributes' => [
+					'id'    => 'general',
+					'title' => __( 'General', 'woocommerce' ),
+				],
+			]
+		);
+		// Basic Details Section.
+		$basic_details = $general_group->add_section(
+			[
+				'id'         => 'basic-details',
+				'attributes' => [
+					'title'       => __( 'Basic details', 'woocommerce' ),
+					'description' => __( 'This info will be displayed on the product page, category pages, social media, and search results.', 'woocommerce' ),
+				],
+			]
+		);
+		$basic_details->add_block(
+			[
+				'id'         => 'product-name',
+				'blockName'  => 'woocommerce/product-name-field',
+				'attributes' => [
+					'name'      => 'Product name',
+					'autoFocus' => true,
+				],
+			]
+		);
+		$basic_details->add_block(
+			[
+				'id'        => 'product-summary',
+				'blockName' => 'woocommerce/product-summary-field',
+			]
+		);
+		$pricing_columns  = $basic_details->add_block(
+			[
+				'id'        => 'product-pricing-columns',
+				'blockName' => 'core/columns',
+			]
+		);
+		$pricing_column_1 = $pricing_columns->add_block(
+			[
+				'id'         => 'product-pricing-column-1',
+				'blockName'  => 'core/column',
+				'attributes' => [
+					'templateLock' => 'all',
+				],
+			]
+		);
+		$pricing_column_1->add_block(
+			[
+				'id'         => 'product-regular-price',
+				'blockName'  => 'woocommerce/product-regular-price-field',
+				'attributes' => [
+					'name'  => 'regular_price',
+					'label' => __( 'List price', 'woocommerce' ),
+					'help'  => __( 'Manage more settings in <PricingTab>Pricing.</PricingTab>', 'woocommerce' ),
```

Comment:
We should have a note for translators so they understand what `<PricingTab>` is and that they shouldn't translate that tag.